incubator/virtualcluster/pkg/syncer/resources/pod/uws.go

```diff
@@ -172,13 +172,30 @@ func (c *controller) backPopulate(key string) error {
 		}
 	}
 
+	spec, err := c.multiClusterPodController.GetSpec(clusterName)
+	if err != nil {
+		return err
+	}
+
+	var newPod *v1.Pod
+	updatedMeta := conversion.Equality(spec).CheckUWObjectMetaEquality(&pPod.ObjectMeta, &vPod.ObjectMeta)
+	if updatedMeta != nil {
+		newPod = vPod.DeepCopy()
+		newPod.ObjectMeta = *updatedMeta
+		if _, err = tenantClient.CoreV1().Pods(vPod.Namespace).Update(newPod); err != nil {
+			return fmt.Errorf("failed to back populate pod %s/%s meta update for cluster %s: %v", vPod.Namespace, vPod.Name, clusterName, err)
```

Comment:
vPod is updated after this operation success.  if you try to update the vPod again，you will get a `has been modify` error because the vPod is outdated. So I add the obj to the queue, and wait for the next loop, next loop will get the fresh one.

Reply:
I add a refetch call before next update.